components/brave_wallet_ui/common/slices/endpoints/transaction.endpoints.ts

```diff
@@ -1769,10 +1785,10 @@ export const transactionEndpoints = ({
             throw new Error(errorMessage)
           }
 
-          const priorityFee = (BigInt(fee.computeUnits)
-                            * BigInt(fee.feePerComputeUnit))
-            / BigInt(BraveWallet.MICRO_LAMPORTS_PER_LAMPORT);
-          const totalFee = BigInt(fee.baseFee) + priorityFee;
+          const priorityFee =
+            (BigInt(fee.computeUnits) * BigInt(fee.feePerComputeUnit)) /
+            BigInt(BraveWallet.MICRO_LAMPORTS_PER_LAMPORT)
+          const totalFee = BigInt(fee.baseFee) + priorityFee
```

Comment:
Is it better to use `Amount` here?

Reply:
This is just formatting changes. I did not author this logic